### DISCUSSION TITLE
Can heat pumps link with PV to optimise local electricity production? Can the AI decide on best approaches?

### DISCUSSION THREAD

**Discussion by samyoung-dsci** [alan-turing-institute/ADViCE#98] [Key sector: Built Environment]
Can heat pumps link with PV to optimise local electricity production? Can the AI decide on best approaches?

**Comment by esc-data-science** [comment]
In short, yes! This will often involve having a Home Energy Management System in the middle to optimise everything, rather than just directly linking the heat pump to the PV.
